.aspect/cli/rules_cc.star

```diff
@@ -0,0 +1,95 @@
+aspect.register_rule_kind("cc_binary", {
```

Comment:
I've not dug into Aspect CLI that deeply yet, so I don't know what these bits are for.

Reply:
Ah, I see the comment in Slack now... this is likely part of the new thing for your upcoming talk, so it makes sense that I haven't heard of this stuff yet.

Reply:
yeah this is "what if you could write a Gazelle extension in a single interpreted script that lives in your repo".

You can see how it already handles all the logic needed here. We'll have a better story for sharing them at some point, but I'm betting the rules_cc maintainers will reject a PR adding it.

Reply:
Could be challenging to get rule maintainers broadly to be interested in accepting this kind of thing. Maybe each could be packaged in a module, placed in to the BCR? Then all the version matching and discoverability is outsourced there.

Reply:
That is possible. We don't even have the ability to `load` from external starlark files in our Gazelle-hosted starlark interpreter yet, so it's still an open design problem.